ir/ir.def

```diff
@@ -205,6 +205,25 @@ class Key : PropertyValue {
     validate { keyElements->check_null(); }
 }
 
+/// Pre-defined entry in a table
+class Entry : IAnnotated {
+    Annotations         annotations;  /// annotations are optional (supported annotations: @priority(value))
+    ListExpression      keys;         /// must be a tuple expression
```

Comment:
default should be a legal key; I suggest having this be an Expression, exactly like a SelectCase.